libmamba/include/mamba/core/channel.hpp

```diff
@@ -18,6 +18,14 @@
 
 namespace mamba
 {
+    // ATTENTION names with substrings need to go longer -> smalle
+    // otherwise linux-ppc64 matches for linux-ppc64le etc!
+    const std::vector<std::string> KNOWN_PLATFORMS = {
```

Comment:
I think it would be better to keep the definition of this variable in the cpp file, and provide a function to get it in the header.

Reply:
OK! I don't understand what difference that makes but not a C++ expert

Reply:
If in the future we want to update the `KNOWN_PLATFORMS` variable:
- if `KNOWN_PLATFORMS` is defined in the header, any binary using it will have to recompile when updating, meaning we break the ABI compatibility.
- if it is defined in a source file, the client can simply upgrade the dependency with no further action required on its side.

I know this header is not really a "public" API yet (in that it's not in `include/api`), but it will likely become one after the ongoing refactoring. We want libmamba to be usable as a framework and not just be a geared towards the CLI.